spec/integration/_dev/benchmark/spec.yml

```diff
@@ -6,12 +6,32 @@ spec:
       type: folder
       name: system
       required: false
-      additionalContents: false
+      additionalContents: true
       contents:
         - description: Folder containing deployable services for benchmarks.
           type: folder
           name: deploy
-          $ref: "./deploy/spec.yml"
+          additionalContents: false
+          required: false
+          contents:
+            - description: Folder containing deployment files for a Docker-based service deployment
+              type: folder
+              name: docker
+              $ref: "./deploy/docker/spec.yml"
+            - description: Folder containing deployment files for a Terraform-based service deployment
+              type: folder
+              name: tf
+              $ref: "./deploy/tf/spec.yml"
+            - description: Folder containing deployment files for a Kubernetes service deployment
+              type: folder
+              name: k8s
+              $ref: "./deploy/k8s/spec.yml"
```

Comment:
This definition also looks like to the one used in `_dev/deploy/k8s`. 
Could it be used this file in the `$ref` key ? 
https://github.com/elastic/package-spec/blob/main/spec/integration/_dev/deploy/k8s/spec.yml